main/src/hadd.cxx

```diff
@@ -1,46 +1,49 @@
-/*
-
-  This program will add histograms (see note) and Trees from a list of root files and write them
-  to a target root file. The target file is newly created and must not be
+/**
+  \file hadd.cxx
+  \brief This program will add histograms (see note) and Trees from a list of root files and write them to a target root file.
+  The target file is newly created and must not be
   identical to one of the source files.
 
   Syntax:
-
+  ```{.cpp}
```

Comment:
I believe the doxygen syntax is
```
~~~{.cpp}
code
~~~
```

Reply:
Doxygen happily accepts both.
Another one you can use is
```
\code
...
\endcode
```
I just use the triple quotes, because it's the same as in github.